package.json

```diff
@@ -14,38 +14,49 @@
     "react-native",
     "driver"
   ],
+  "files": [
+    "lib"
```

Comment:
should also include `LICENSE` and `readme.md`

Reply:
LICENCE and readme are included by default: https://docs.npmjs.com/files/package.json#files, that's why `files` is a more appropriate way to include than `.npmignore`, files also support negate patterns.